contracts/contracts/ProviderRegistry.sol

```diff
@@ -201,13 +198,10 @@ contract ProviderRegistry is
         emit WithdrawalDelayUpdated(_withdrawalDelay);
     }
 
-    /**
-     * @dev Reward funds to the fee receipt.
-     */
-    function withdrawFeeRecipientAmount() external nonReentrant {
-        feeRecipientAmount = 0;
-        (bool successFee, ) = feeRecipient.call{value: feeRecipientAmount}("");
-        require(successFee, "fee recipient transfer failed");
+    /// @dev Sets the fee payout period in blocks
+    /// @param _feePayoutPeriodBlocks The new fee payout period in blocks
+    function setFeePayoutPeriodBlocks(uint256 _feePayoutPeriodBlocks) external onlyOwner {
+        protocolFeeTracker.payoutPeriodBlocks = _feePayoutPeriodBlocks;
```

Comment:
It's a good practice to generate event in case that function is called
```suggestion
         protocolFeeTracker.payoutPeriodBlocks = _feePayoutPeriodBlocks;
         emit FeePayoutPeriodBlocksUpdated(_feePayoutPeriodBlocks);
```